### ISSUE TITLE
`write_csv()` does not support specifying the decimal separator

### ISSUE THREAD

**Issue by matthias-Q** [pola-rs/polars#9501]
### Problem description

Pandas has the option to set the decimal separator. In some countries it is common to use `.` as a separator for 1000 digits and `,` for the decimal separator. e.g. `10.000,05` (spoken: ten thousand point zero five)

Additionally, it is currently not possible to specify the encoding of the file.

**Comment by matthias-Q** [comment]
I will try to submit a PR for that.

**Comment by ritchie46** [comment]
We wouldn't be able to read such a file: #6698

**Comment by matthias-Q** [comment]
`lexical_core` supports setting the format options and seems to be already used. I assume that it would work with the `read_csv` as well. There needs to be options to specify the characters.

In Pandas you can specify them as well.

**Comment by ritchie46** [comment]
Yes, but we first should fix the reading before we add the writing is what I meant. :)

**Comment by igmriegel** [comment]
Hello guys, how is the work on this issue going on?

I'm waiting the development of this feature as stated on the issue https://github.com/pola-rs/polars/issues/6698

Is there something on the Python level of the code where I can help?

**Comment by dmoklaf** [comment]
That one is a showstopper for Polars in latin  countries (France, Spain, Italy, ...)

**Comment by rgeronimi** [comment]
Any intent to resolve this? It's impossible to work fluently with CSV files from any Latin country without this

**Comment by ritchie46** [comment]
@kdn36 can you do this one?
